Sources/MapboxNavigation/VisualInstruction.swift

```diff
@@ -108,6 +108,33 @@ extension VisualInstruction {
         return CPImageSet(lightContentImage: maneuverIcons[1], darkContentImage: maneuverIcons[0])
     }
     
+    /**
+     Returns a `UIImage` representing the maneuver.
+     
+     - parameter side: Driving side of the road cars and traffic flow.
+     - parameter userInterfaceStyle: The `UIUserInterfaceStyle` that the maneuver will be displayed in.
+     
+     - returns: A `UIImage` representing the maneuver.
+     */
+    @available(iOS 13.0, *)
+    public func maneuverImage(side: DrivingSide, userInterfaceStyle: UIUserInterfaceStyle) -> UIImage? {
+        var color: UIColor
```

Comment:
This can be declared as `let` I think:
```suggestion
        let color: UIColor
```

Also should we expose this function publicly right away? IMHO it's always better to make it internal first and if there is such need make it public afterwards.

Reply:
I see. Thanks for the suggestion, I've removed the public mark and updated the changeLog.